sound/soc/codecs/rt1308-sdw.c

```diff
@@ -561,13 +561,19 @@ static int rt1308_sdw_hw_params(struct snd_pcm_substream *substream,
 		return -EINVAL;
 	}
 
+	num_channels = params_channels(params);
+	ch_mask = (1 << (num_channels)) - 1;
```

Comment:
superfluous inner parentheses. You could also use `BIT()` or even `GENMASK()`

Reply:
removed parentheses.